### ISSUE TITLE
Proposal: Rounded corner for Image

### ISSUE THREAD

**Issue by harvinders** [microsoft/microsoft-ui-xaml#1261]
# Proposal: Rounded corner for Image

## Summary

Since we are updating all the controls with `CornerRadius`, should we also not add it to the `Image` Control. 

## Rationale

* Sharp edge of the `Image`Control in an app full of rounded corners controls would probably look odd.

## Open Questions

Should GridView Items also have rounded corners?

**Comment by mdtauk** [comment]
By default I would say the Image, should be the Image, without cropping or corner radius.

But the corner radius should work on the control.

**Comment by jevansaks** [comment]
Image is a primitive element, it shouldn't be rounded by default. But it's probably reasonable to have rounded corner properties on the element.

**Comment by harvinders** [comment]
@mdtauk , @jevansaks I agree, I have updated the proposal, clarifying rounded corner requirement for Image control.

**Comment by chigy** [comment]
@harvinders , to your question "Should GridView Items also have rounded corners?" 
GridViews and ListViews were not rounded on purpose. The way our design team sees the rounded corner is that outside elements should be rounded but not inner contents.
